### PR TITLE
Remove node sass

### PR DESCRIPTION
**Pull request by 0xClpz**

### CODE REVIEW

File: packages/inab-web/package.json [hunk 1]
```diff
@@ -45,7 +45,6 @@
     "eslint-plugin-react": "^7.7.0",
     "flow-bin": "^0.71.0",
     "lint-staged": "^7.0.5",
-    "node-sass": "^4.9.0",
```
Comment (hwaterke) [inline packages/inab-web/package.json:48]: `sass-loader` can go too ;)

File: packages/inab-web/src/styles/tables.scss [hunk 2]
```diff
@@ -1,6 +0,0 @@
-.table > tbody > tr > td.right,
```
Comment (hwaterke) [inline packages/inab-web/src/styles/tables.scss:1]: Did you replace the `td.right` and `th.right`? :)

Comment (0xClpz) [reply packages/inab-web/src/styles/tables.scss:1]: the .right class wasn't used anywhere lol

Comment (0xClpz) [reply packages/inab-web/src/styles/tables.scss:1]: Oh wow nevermind it's actually used !

**Review by hwaterke — APPROVED** [review]

**Pull request merged**